ivy.xml

```diff
@@ -79,5 +80,8 @@
 
   <dependency org="io.fusionauth" name="fusionauth-java-client" rev="1.42.0" />
 
+  <dependency org="org.eclipse.jetty" name="jetty-webapp" rev="9.4.6.v20170531" conf="deploy->default" />
```

Comment:
When we run devmode, we currently get these APIs on our classpath from gwt-devmode.jar, but when running Scalatra without either this or Tomcat they need to be supplied manually